src/Inkluzitron/Services/PointsService.cs

```diff
@@ -222,19 +251,21 @@ public async Task<TemporaryFile> GetPointsAsync(IUser user)
 
             using var profilePicture = await ImagesService.GetAvatarAsync(user);
 
+            var ppDominantColor = profilePicture.Frames[0].GetDominantColor();
```

Comment:
Napadá mě zde ještě vylepšení, když má někdo animovanou profilovku, tak to tu dominantní barvu bude počítat z každého frame, takže se ta barva bude měnit s běžící profilovkou.

Reply:
Spíš ne, to by bylo rušivý AF

Reply:
Držím se toho, jak to dělá Discord